src/chainparams.cpp

```diff
@@ -218,14 +218,13 @@ class CMainParams : public CChainParams {
         assert(genesis.hashMerkleRoot == uint256S("0x72ddd9496b004221ed0557358846d9248ecd4c440ebd28ed901efc18757d0fad"));
 
         // Note that of those with the service bits flag, most only support a subset of possible options
-        vSeeds.emplace_back("seed1.digibyte.io");
-        vSeeds.emplace_back("seed2.digibyte.io");
-        vSeeds.emplace_back("seed3.digibyte.io");
-        vSeeds.emplace_back("seed.digibyte.io");
-        vSeeds.emplace_back("digihash.co");
-        vSeeds.emplace_back("digiexplorer.info");
-        vSeeds.emplace_back("seed.digibyteprojects.com");
-
+        vSeeds.emplace_back("seed.digibyte.org"); // Website collective
+        vSeeds.emplace_back("seed.digibyteservers.io"); // ChillingSilence
+	vSeeds.emplace_back("seed.digibyte.io"); // JaredTate
+	vSeeds.emplace_back("seed.digibyteblockchain.com"); // JS555
```

Comment:
Seems to be offline

Reply:
seed.digibyte.io, seed.digiexplorer.info & seed.digibytefoundation.org should be left in. We can get peer seeder nodes on all those domains. Not sure who runs digibyteblockchain.com. Is not me.